TM1py/Utils/Utils.py

```diff
@@ -302,6 +304,10 @@ def build_content_from_cellset_dict(
                 coordinate = extract_unique_names_from_members(axis['Tuples'][tuple_ordinal]['Members'])
                 coordinates.extend(coordinate)
 
+                if skip_sandbox_dimension:
+                    coordinates = [element for element in coordinates if
+                                   "[sandboxes].[sandboxes].[" not in element.lower()]
+
         coordinates = sort_coordinates(cube_dimensions, coordinates, element_unique_names=element_unique_names)
```

Comment:
Thanks for the feature @cubewise-gng!
This will be helpful.

I am sure it works, but a bit fearful to include another string operation so deep in the loop. This line gets executed for every cell in a result set so potentially 100k times for an MDX execution.

I wonder if there might be a "cheaper" way to drop the sandbox from the coordinates.
If we drop the first entry in the `cube_dimensions` list would that produce the desired outcome in the `sort_coordinates` function?
I would prefer to use some kind of shortcut for that operation